core/src/main/java/edu/wpi/grip/core/util/OpenCVUtility.java

```diff
@@ -0,0 +1,39 @@
+package edu.wpi.grip.core.util;
+
+
+import com.google.common.eventbus.EventBus;
+import org.bytedeco.javacpp.opencv_core.Mat;
+import org.bytedeco.javacpp.opencv_imgcodecs;
+
+import java.io.IOException;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * A utility wrapper for running specific OpenCV method in the cleanest safest ways that we know.
+ */
+public final class OpenCVUtility {
```

Comment:
Fine, whatever. I don't like that because if we add more methods then we'd have to create another utility.